browser.js

```diff
@@ -26,17 +26,19 @@ Updates should happen there first.
 */
 const scrollToTweet = event => { // eslint-disable-line
 	const $ = document.querySelector.bind(document);
-	const navHeight = $('nav').clientHeight;
 	const tweets = document.querySelectorAll('._222QxFjc[role="row"]');
 	const currentTop = window.scrollY;
 	const bufferTop = 5; // added because somtimes the scrolling is off by a few px
 	const keyCode = event.charCode; // TODO: replace with key when Chrome 51 is out https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key
 
+	// calculate the height of the navigation/header to use for scrolling
+	const calculatedNavHeight = $('header').clientHeight + ($('header').clientHeight - $('nav').clientHeight);
+
 	// this padding has to be applied to factor in the padding that's added to the container
 	const tweetsContainerPadding = Math.ceil(parseInt($('._1nQuzuNK').style.paddingTop.replace('px', ''), 10));
 
 	// shortcut for getting the total offset from the top of the document of a particular tweet
-	const totalOffset = tweetOffset => tweetOffset + navHeight + tweetsContainerPadding;
+	const totalOffset = tweetOffset => tweetOffset + calculatedNavHeight + tweetsContainerPadding;
```

Comment:
That looks right, yup. Just to explain the logic, the `totalOffset` is "the mark we will scroll the page to when you press `j` or `k`". The `tweetIsBelowNav` is "the imaginary line a tweet has to cross to be considered 'the next tweet'".

If you increase the number in `totalOffset` you scroll further on a keypress. If you increase the number in `tweetIsBelowNav` you move the imaginary line down further.

The top of the "next tweet" (first tweet below the line) becomes the "mark" for scrolling when you type <kbd>j</kbd>

The top of the tweet two above the "next tweet" for scrolling when you type <kbd>k</kbd>